src/components/chips/js/chipsController.js

```diff
@@ -170,18 +177,40 @@ MdChipsCtrl.prototype.init = function() {
   var ctrl = this;
 
   // Set the wrapper ID
-  ctrl.wrapperId = '_md-chips-wrapper-' + ctrl.$mdUtil.nextUid();
+  this.wrapperId = '_md-chips-wrapper-' + this.$mdUtil.nextUid();
```

Comment:
What's the benefit of the changes on this line?

Reply:
Actually I do not see a point in using a helper-var instead of `this` if `this` is usable in a place. In my opinion it reduces the readability of code. And if the helper-var for encapsulated inner functions is called `ctrl`, `self` or `that` is in my opinion too much a matter of taste (otherwise I would have replaced `ctrl` with more widely used helpers in this repository)

Reply:
Ah OK. 👍